### DISCUSSION TITLE
Foxglove desktop link with remote file does not open remote file immediately

### DISCUSSION THREAD

**Discussion by bernard-bear** [foxglove/community#999] [Bugs]
**Version:**
2.3.0
Desktop app

**Steps to reproduce:**
(1) Create a shareable link with as such:
`https://app.foxglove.dev/view?ds=remote-file&ds.url={REMOTE_ROSBAG_FILE}&openIn=desktop`

Note: Same thing happens if I add in the `{YOUR-FOXGLOVE-SLUG}`

(2) Open the link.

(3) Observe what happens when the desktop app opens.

**Expected Behaviour:**
Desktop app loads the remote file and navigates to the page with the layout, so that the user can immediately start interacting with the rosbag via Foxglove Studio.

**Actual behaviour:**
Desktop app stays on the dashboard page. Only if I click on "Open connection", then the remote file is loaded. This is bad UX because users might think that the file was not successfully loaded.

**Comment by jtbandes** [comment]
This is already fixed in 2.4.0 😄
